---
content_type: page
description: This page contains information on the readings, in-lab questions, and
  pre-lab and post-lab assignments for this day of the lab.
draft: false
title: Experiment 5, Lab Day 19
uid: 874870c4-26af-4c91-becc-9dbd89e436c6
---
{{< tableopen >}}{{< theadopen >}}{{< tropen >}}{{< thopen >}}
Parts
{{< thclose >}}{{< thopen >}}
Pages in Lab Manual
{{< thclose >}}{{< trclose >}}{{< theadclose >}}{{< tbodyopen >}}{{< tropen >}}{{< tdopen >}}
Part 5.1: Design SARS-CoV-2 target protein
{{< tdclose >}}{{< tdopen >}}
107–114
{{< tdclose >}}{{< trclose >}}{{< tbodyclose >}}{{< tableclose >}}

{{% resource_link "6f2f372a-4014-4826-902b-e21e874da375" "Day 19 In-Lab Questions (PDF)" %}}

### Supporting Files

**Note:** The .dna files used in this course contain DNA sequence data that are required to complete the post-lab activities. [SnapGene](https://www.snapgene.com/) is a molecular biology software application that is used to view and work with these files, and is available to [try for free](https://www.snapgene.com/free-trial). 

{{% resource_link "94d96db3-ddba-4ac5-906e-df446a6870ea" "Day 19 Protein Alignment Worksheet (PDF)" %}}

{{% resource_link "61afa891-63cf-405d-9acd-a3387a6cce8f" "229E CoV Genome annotated (.dna)" %}}

{{% resource_link "f911edc3-d96f-47b0-910b-49d14a29748a" "HKU1 CoV Genome annotated (.dna)" %}}

{{% resource_link "c859af01-8e7c-41e7-8871-1426d127fb03" "NL63 CoV Genome annotated (.dna)" %}}

{{% resource_link "7ca353f2-01b7-4117-b86c-4605a02795bf" "OC43 CoV Genome annotated (.dna)" %}}

{{% resource_link "b406a197-2bd3-482d-ad8c-0fefbe1fbe66" "pET-45b Plasmid (.dna)" %}}

{{% resource_link "f7047045-2af4-463e-afe3-e79ce89717ea" "SARS-2 Omicron BA.2 Complete Genome annotated (.dna)" %}}

{{% resource_link "3b3ec0f3-ff82-409a-9f1c-c1e0e71b2bb7" "SARS-2 Original Complete Genome annotated (.dna)" %}}

(Source: National Library of Medicine, accessed in Fall 2022.)

[Pre-Lab & Post-Lab Instructions](https://draft.ocw.mit.edu/courses/7-003-applied-molecular-biology-lab-spring-2022/pages/lab-notebook-instructions/)

## Day 19 Post-Lab Notebook Tips and Reminders

Follow the instructions and answer all nine questions below to complete your Day 19 post-lab on LabArchives:

1. Which SARS-CoV-2 protein do you choose as your target viral protein (pick from either the E, M, N, or S proteins)?
2. Will you be using the full-length protein as a target or will you only be using a portion of the protein as a target? If you will only be using a particular region of the protein, list which basepairs of the gene sequence or which amino acid residues of the protein sequence you will be using.
3. Perform a multiple protein alignment that aligns your SARS-CoV-2 target protein with the corresponding homologous protein from all of the four common human coronaviruses (229E, NL63, OC43, and HKU1). Take a screenshot of the protein alignment (focusing on the region of interest within your target protein, if applicable). Insert a JPG of that protein alignment screenshot in your Day 19 post-lab on LabArchives.                    
    Detailed instructions for performing a multiple protein alignment (using either SnapGene or [NCBI COBALT](https://www.ncbi.nlm.nih.gov/tools/cobalt/re_cobalt.cgi)) are provided here: {{% resource_link "94d96db3-ddba-4ac5-906e-df446a6870ea" "Day 19 Protein Alignment Worksheet (PDF)" %}}
4. Briefly explain your answers to Questions 1 and 2: Why did you choose that particular protein (and protein region) to use as your target viral protein for engineering antibodies that can detect SARS-CoV-2 virus?
5. Design PCR primers to amplify your SARS-CoV-2 target gene (region) for cloning into the pET-45b expression plasmid such that your SARS-CoV-2 protein can be expressed from the plasmid’s T7lacO promoter with a 6xHis-tag (for purification of the protein later).
    1. Write your forward and reverse primer sequences, labelling the 5’ and 3’ ends of each primer.
    2. Underline any parts of your primer sequences that bind to any SARS-CoV-2 gene sequence.
    3. List any restriction enzyme sites present in your primers and use a different color font or color highlight to indicate where the restriction sites are in your primer sequences.
    4. List the T<sub>m</sub> (melting temperature) for each primer. (SnapGene will calculate approximate T<sub>m</sub>s for primers, or you can use the NEB online T<sub>m</sub> calculator at [https://tmcalculator.neb.com](https://tmcalculator.neb.com/#!/main) )
6. List the length of your expected PCR product (in basepairs).
7. Write a PCR program to amplify your SARS-CoV-2 target gene (region) using your primers from Question 5. List the temperatures and times of all steps in your PCR thermocycler program.
8. List what restriction enzymes you will use to digest your PCR product (target gene insert) and what restriction enzymes you will use to digest the pET-45b plasmid for cloning.
9. Create a SnapGene .dna file consisting of your SARS-CoV-2 target gene region inserted into the pET-45b vector, following your cloning strategy from Question 8.
    1. Annotate or label the SARS-CoV-2 gene sequence in the plasmid as a new feature (highlight the desired sequence, then from the top menu bar, go to Features → Add Feature…).
    2. Add both your forward and reverse PCR primers from Question 5 (from the top menu bar, go to Primers → Add Primer…).
    3. Save the .dna file as “pET-SARS2” with your initials and protein name (e.g., “VC pET-SARS2 M protein.dna”).
    4. Upload this pET-SARS2 .dna file as an attachment in your Day 19 Post-Lab on LabArchives.